traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -78,7 +78,10 @@
                         <option ng-value="false">Not Active</option>
                     </select>
                     <small class="input-error" ng-show="hasPropertyError(deliveryServiceForm.active, 'required')">Required</small>
-                    <small class="input-diff" ng-if="settings.isRequest" ng-show="open() && deliveryService.active != dsCurrent.active">Current Value: [ {{dsCurrent.active ? 'Active' : 'Not Active'}} ]</small>
+                    <fieldset class="current-value" ng-if="settings.isRequest" ng-show="open() && deliveryService.active != dsCurrent.active">
+                        <legend>Current Value</legend>
+                        <input type="text" class="form-control" readonly value="{{dsCurrent.active ? 'Active' : 'Not Active'}}">
```

Comment:
"The <output> tag represents the result of a calculation (like one performed by a script)."

that's a definition I found so not sure output is the right answer but i did try it and i was getting weird wrapping issues if the value was long.
 
as far as styling, i was just looking for the look of a disabled input/textarea and those elements give me that by simply using the readonly attribute. whereas, if i wanted to use small i'd have to wrestle with some styling. 

and yeah, the fieldset, though probably not technically the correct element, was simply used to achieve the style/layout.

Reply:
I'm not sold on `<textarea>` either, since it's defined as an editing control that gets submitted along with the parent form. Not that we actually do form submission, but semantically user-agents will expect that it is an input control that simply cannot be changed. You could try `<pre>` or `<samp>` instead.

Technically anything that would be a previous value is the result of some user's actions, but not necessarily recently, or by the current user, or even using the TP UI at all. Plus, a previous value can only exist if there's a new current value. So maybe `<output>` isn't the best way to represent it, but I could see an argument for it.

Styling of a fieldset appear consistent across Chrome, Firefox and Safari so there doesn't really seem to be any harm in using it (unless you want its contents to display with `flex` or `grid`, which is bugged in Edge and Chrome). My only concern is that it appears to the user-agent as a set of grouped controls, and actually does contain `<input>`/`<textarea>` form controls that aren't really part of the form.

Reply:
let me try pre or samp